app/models/company.rb

```diff
@@ -26,4 +28,22 @@ def self.company_count_by_status(status)
   def self.company_count_by_city
     group(:city).count.sort_by { |_key, value| value }.pop(10).reverse
   end
+
+  private
+
+  def create_new_user
+    password = User.generate_password
+    company = Company.last
+    user = User.create(email: company.email,
+                       name: company.name,
+                       title: company.title,
+                       password: password,
+                       company_id: company.id
+                       )
+    email_new_user(user, password)
```

Comment:
I could have created another after_create for emailing the new user, but since emailing that user relies on creating the user I just called it at the end of this method.